resources/apigateway/helm-values.yaml

```diff
@@ -27,12 +36,6 @@ plugins:
   - pluginName: phantom-token
     name: curity-phantom-token
 
-#
```

Comment:
This is the default certificate and it is seems recommended to reference secrets (that might be created by cert-manager) in YAML resources rather than overriding the default certificate here.